frontend/integration-tests/tests/performance.scenario.ts

```diff
@@ -66,7 +66,7 @@ describe('Performance test', () => {
     const initialChunks = await browser.executeScript<{name: string, size: number}[]>(() => performance.getEntriesByType('resource')
       .filter(({name}) => name.endsWith('.js')));
 
-    await crudView.selectOptionFromGear('console', 'Edit Deployment');
+    await crudView.selectOptionFromGear('openshift-console', 'Edit Deployment');
```

Comment:
I'd suggest switching to a resource that is there for both 3.11 and 4.0 to keep things simple. Maybe config map `console-config` in the `openshift-console` namespace?

Reply:
We won't be able to merge the initial PR without this change as well since it will fail CI against our existing shared cluster.